fusor-ember-cli/app/routes/deployment.js

```diff
@@ -43,6 +44,62 @@ export default Ember.Route.extend(DeploymentRouteMixin, {
 
   },
 
+  loadOpenStack(controller, model) {
+    var self = this;
+    if (model.get('deploy_openstack') && !Ember.isBlank(model.get('openstack_undercloud_password'))) {
```

Comment:
@cfchase, I think we should only call `loadOpenStack` after checking that Satellite and RHEV are valid. If RHEV isn't valid, then openstack tab should be disabled anyway.